README.md

```diff
@@ -7,256 +7,317 @@
-        Open {% else %} Closed {% endif %}
-      title_offset: 10
+```yaml
+- page_type: Fuel
+  enabled: "{{ states.input_boolean.YOURS.state }}"
+  title: Classic
+  name1: Diesel
+  price1: "{{ states.sensor.diesel.state }}"
+  name2: Super
+  price2: "{{ states.sensor.super.state }}"
+  name3: E10
+  price3: "{{ states.sensor.e10.state }}"
+  status: >-
+    {% if is_state('binary_sensor.status', 'on') %} Offen {%
+    else %} Geschlossen {% endif %}
+  title_offset: "10"
 ```
 
 <br>
 
 --------------
 
-<br>
+### channel 
+*In Divoom app you can set three different custom channels which you can select here.*
 
+| **Keywords**         | **Values**                                   |
+|:---------------------|:---------------------------------------------|
+| - page_type: channel | to use this page_type                        |
+| id                   | 0 = channel 1, 1 = channel 2, 2 = channel 3  |
 
-### Custom Channel - In Divoom app you can set three different custom channels which you can select here. 
-| **Keywords**    | **Values**                                   |
-|:----------------|:---------------------------------------------|
-| channel         | Custom Channel in APP                        |
-| number          | 0 = channel 1, 1 = channel 2, 2 = channel 3  |
+```yaml
+- page_type: channel
+  id: 0 #shows custom channel 1
+```
 
 <br>
 
 --------------
 
 <br>
 
-### ClockId Channel - In Divoom app you can set three different custom channels which you can select here. 
-| **Keywords** | **Values**         |
-|:-------------|:-------------------|
-| clockId      | Clock Faces in APP |
-| number       | int                |
+### clock
+*Channel - In Divoom app you can set three different custom channels which you can select here.*
+
+| **Keywords**       | **Values**             |
+|:-------------------|:-----------------------|
+| - page_type: clock | to use this page_type  |
+| id                 | int                    |
 
-If you have any further questions, I will be happy to help.
+```yaml
+- page_type: clock
+  id: 182
+```
 
-### To find out the number of the ClockFace, you can proceed as follows.
+*To find out the number of the ClockFace, you can proceed as follows.*
 1. First go to the settings and activate debug logging in the Divoom Pixoo 64 integration.
 2. Now open the Divoom app on your smartphone and select your preferred ClockFace.
 3. As soon as this is displayed on your Pixoo64, you will find "Device Data" in the log and then "CurClockId".
 4. The CurClockId is the number you were looking for.
 
-
-
-
 <br>
 
 --------------
 
-<br>
 
-## Automation Example
 
-You can use it for Push Notifications.
-Trigger with anything!
+## Automations as Push Notifications
 
-Action with the Service "Divoom Pixoo 64: Use for push notifications".
+You can use it for Push Notifications. Trigger with anything! Action with the Service "Divoom Pixoo 64: Use for push notifications".
```

Comment:
I renamed the service to "Divoom Pixoo 64: Send a page to Divoom Pixoo". I think it makes more sense because of the new usage, but using it as a push notification is still indeed possible.